Update environmental-sensors.md

rfraposa opened the pull request:
The instructions use `s3Cluster`, but `S3Queue` is probably a better option. It's faster, and you don't have to worry about network timeouts during ingestion.

### Changelog category (leave one):
- Documentation (changelog entry is not required)


> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

#### CI Settings (Only check the boxes if you know what you are doing):
- [ ] <!---ci_set_required--> Allow: All Required Checks
- [ ] <!---ci_include_stateless--> Allow: Stateless tests
- [ ] <!---ci_include_stateful--> Allow: Stateful tests
- [ ] <!---ci_include_integration--> Allow: Integration Tests
- [ ] <!---ci_include_performance--> Allow: Performance tests
- [ ] <!---ci_set_builds--> Allow: All Builds
- [ ] <!---batch_0_1--> Allow: batch 1, 2 for multi-batch jobs
- [ ] <!---batch_2_3--> Allow: batch 3, 4, 5, 6 for multi-batch jobs
---
- [ ] <!---ci_exclude_style--> Exclude: Style check
- [ ] <!---ci_exclude_fast--> Exclude: Fast test
- [ ] <!---ci_exclude_asan--> Exclude: All with ASAN
- [ ] <!---ci_exclude_tsan|msan|ubsan|coverage--> Exclude: All with TSAN, MSAN, UBSAN, Coverage
- [ ] <!---ci_exclude_aarch64|release|debug--> Exclude: All with aarch64, release, debug
---
- [ ] <!---ci_include_fuzzer--> Run only fuzzers related jobs (libFuzzer fuzzers, AST fuzzers, etc.)
- [ ] <!---ci_exclude_ast--> Exclude: AST fuzzers
---
- [ ] <!---do_not_test--> Do not test
- [ ] <!---woolen_wolfdog--> Woolen Wolfdog
- [ ] <!---upload_all--> Upload binaries for special builds
- [ ] <!---no_merge_commit--> Disable merge-commit
- [ ] <!---no_ci_cache--> Disable CI cache

rfraposa committed:
Update environmental-sensors.md

The instructions use `s3Cluster`, but `S3Queue` is probably a better option. It's faster, and you don't have to worry about network timeouts during ingestion.

robot-ch-test-poll3 commented:
<!-- automatic status comment for PR #69803 from ClickHouse/ClickHouse:rfraposa-patch-2 -->
*This is an automated comment for commit 24413a5c77b63d3d74d3f97c024610e936b53083 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/69803/24413a5c77b63d3d74d3f97c024610e936b53083/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Style check</td><td>Runs a set of checks to keep the code style clean. If some of tests failed, see the related log from the report</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/69803/24413a5c77b63d3d74d3f97c024610e936b53083/style_check.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

rfraposa committed:
Update environmental-sensors.md

I think setting `s3queue_processing_threads_num = 32` was way too many threads - it stopped loading the data about 75% of the way through. Trying again with 3 threads

rfraposa commented:
Leaving as is. The synchronous ingest works fine and is faster than the S3Queue method
